script/vsts/platforms/linux.yml

```diff
@@ -8,6 +8,9 @@ jobs:
       vmImage: ubuntu-16.04
 
     steps:
+      - script: which npm
+        displayName: Check npm location
```

Comment:
Do we need this here?

Reply:
I think I was just testing which npm versions are being used because of how Linux isn't using the expected one, and this PR makes mac not use the expected one. It can be removed. I'm not interested in 'fixing' the wrong npm usage though.